README.md

```diff
@@ -72,6 +72,8 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 - [Cryptos website](https://cryptos-javivy.netlify.app/)
+- [Video](https://www.loom.com/share/f715262d258245a1b5cc2ab0da6a3388)
+
 
```

Comment:
- [ ] When I do `npm install` and later run your project locally, it does not compile. Check the screenshot from my and check to ensure it compiles and run locally at your end.
![Screenshot from 2022-12-02 06-00-01](https://user-images.githubusercontent.com/62849872/205219650-37fabdad-fb82-40ae-a0f7-836a1795e3d3.png)

- The errors are mostly coming from linters. Please kindly check and fix them.
- A file like `reportWebVitals.js` is causing errors and I guess that file has no use in your project. You delete the file.

Reply:
Hey kind reviewer :)) @t-yanick , I ran the project in dev branch and create-structure branch and I had no problems in any of them, I'll attach the pictures below, and I don't see that reportWebVitals.js file, so I don't know how could I delete it hehe.

**Create structure branch**
![image](https://user-images.githubusercontent.com/84439052/205223580-2315a34a-4100-416c-8544-6db21d8e2a8e.png)

**dev Branch**
![image](https://user-images.githubusercontent.com/84439052/205222692-e03d1e4b-7f0c-4437-9c58-600504b2cd2f.png)

Reply:
Thanks for checking that and the screenshot.  The next reviewer will sort out things so that you can proceed. 
Happy coding